kalite/distributed/management/commands/setup.py

```diff
@@ -250,11 +269,25 @@ def handle(self, *args, **options):
         call_command("syncdb", interactive=False, verbosity=options.get("verbosity"))
         call_command("migrate", merge=True, verbosity=options.get("verbosity"))
 
+        # download assessment items
         # This can take a long time and lead to Travis stalling. None of this is required for tests.
-        if not settings.RUNNING_IN_TRAVIS:
-            # download assessment items
-            # TODO-BLOCKER (aron): do not hardcode this, and put in the proper URL once we have a redirect set up
-            call_command("unpack_assessment_zip", "http://eslgenie.com/media/assessment_item_resources.zip")
+        if not settings.RUNNING_IN_TRAVIS and options['interactive']:
+            sys.stdout.write("\nStarting in version 0.13, you will need an assessment items package in order to access many of the available exercises.\n")
```

Comment:
I see the value of explicitly saying `sys.(pipe).write`, but I stylistically prefer a `print` function in here.